index.html

```diff
@@ -26,7 +26,7 @@ <h3 class="logo pointer">TESFAYE.T</h3>
 
     <main>
       <section>
-        <h1>I'm Tesfaye <br>Glad to see you!</h1>
+        <h1>I'm Tesfaye <br>Glad to see<br> you!</h1>
         <p>
           I'm a software developer! I can help you build a product , feature
           or website Look through some of my work and experience! If you like
```

Comment:
- [ ] Kindly add your social media link attached to these icons because your portfolio will be seen by many people and they may want to link with you :link: :smile: 
![Screenshot from 2023-01-26 15-47-53](https://user-images.githubusercontent.com/80458199/214852945-35f70cba-fa18-40e9-9b06-8caa5f8997f4.png)
![Screenshot from 2023-01-26 15-47-09](https://user-images.githubusercontent.com/80458199/214852966-dbacd671-317e-4440-8307-d7378ad40915.png)